generated/client/client.go

```diff
@@ -60,8 +63,10 @@ func JoinByFormat(data []string, format string) string {
 	}
 	return strings.Join(data, sep)
 }
+
+// GetBooks returns...
```

Comment:
Maybe `GetBooks makes a <GET/POST/...> request to <URL pattern from swagger spec>. <Description field from swagger spec>`.